lib/components/ContactUsForm.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import {
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  HStack,
+  Input,
+  Center,
+  Box,
+  Button,
+  Textarea,
+} from '@chakra-ui/react';
+
+interface contactFormProps {}
+
+interface NameFormProps {
+  firstNameInput: string;
+  lastNameInput: string;
+  setFirstInput: any;
+  setLastInput: any;
+  isFirstNameMissing: boolean;
```

Comment:
💭 I wonder if this calls for a form validation library

Reply:
good question, it was a bit tedious, if more fields are needed and complex logic is needed then yeah, but rn designs just call for an additional subject field